extensions/guacamole-auth-http/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
While there does need to be a copy of the Apache License included in the `.tar.gz` bundle (see the packaging of the other extensions), there shouldn't be a top-level copy. The guacamole-client source must have exactly one top-level `LICENSE` file, and that's already present.